docs/src/caller.md

```diff
@@ -70,7 +70,33 @@ julia> Singular.LibCentral.center(A, 3)   # base ring cannot be infered from the
 Singular ideal over Singular G-Algebra (QQ),(x,y,z,t),(dp(4),C) with generators (t, 4*x*y + z^2 - 2*z)
 ```
 
-## Global variables
+## Global Interpreter Variables
+
+The function `Singular.call_interpreter` can be used to execute arbitrary
+strings inside the Singular interpreter, and the function
+`Singular.lookup_library_symbol` fetches results from the interpreter.
+
+```@docs
+Singular.lookup_library_symbol(package::String, name::String)
+```
+
+**Examples**
+
+```julia
+julia> Singular.call_interpreter("bigint a = 42;");
+
+julia> a = Singular.lookup_library_symbol("Top", "a"); (a, typeof(a))
+(42, BigInt)
+
+julia> Singular.call_interpreter("ring r=0,(x,y,z),dp; poly f = (x+y)^2;");
+
+julia> Singular.lookup_library_symbol("Top", "r")
+2-element Vector{Any}:
+ Singular Polynomial Ring (QQ),(x,y,z),(dp(3),C)
+ Dict{Symbol, spoly{n_Q}}(:f => x^2 + 2*x*y + y^2)
+```
+
```

Comment:
Great, thanks!!

Maybe move the example into the docstring for `Singular.lookup_library_symbol`? That way it'll be available from the REPL, too.